Resources/Prototypes/Catalog/Fills/Backpacks/backpack.yml

```diff
@@ -0,0 +1,32 @@
+- type: entity
+  parent: ClothingBackpackSyndicate
+  id: ClothingBackpackSyndicateOperative
+  name: operative backpack
+  description: An explosion-proof backpack for holding various nuclear operative goods.
+  components:
+  - type: StorageFill
+    contents:
+      - id: BoxSurvivalSyndicate
+      - id: WeaponPistolViper
+      - id: PinpointerSyndicateNuclear
+      - id: DeathAcidifierImplanter
+
+
```

Comment:
Also having these in 2 separate files without parenting is guaranteed to cause issues down the line.

Reply:
combined into one file